src/djvulibre.mk

```diff
@@ -0,0 +1,29 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := djvulibre
+$(PKG)_IGNORE   := 3.5.27
+$(PKG)_VERSION_ := 3.5.25
+$(PKG)_VERSION  := $($(PKG)_VERSION_).3
+$(PKG)_CHECKSUM := 898d7ed6dd2fa311a521baa95407a91b20a872d80c45e8245442d64f142cb1e0
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION_)
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://sourceforge.net/projects/djvu/files/DjVuLibre/$($(PKG)_VERSION_)/$($(PKG)_FILE)/download
+$(PKG)_DEPS     := gcc jpeg tiff zlib
+
+define $(PKG)_UPDATE
+    $(WGET) -q -O- 'http://sourceforge.net/projects/djvu/files/DjVuLibre/' | \
```

Comment:
How exactly do I have to add the macro? Replacing `https://sourceforge.net` or `sourceforge.net` makes the downloads fail.